gameObjects.py

```diff
@@ -32,10 +32,10 @@ def checkRightBoundary(self, width):
     def draw(self, screen, color):
         position = (self.x, self.y)
```

Comment:
You could type cast it and store in position and pass that around instead of writing the x, y tuples again.

Reply:
True